### PR TITLE
AI can view Entertainment camera

### PR DESCRIPTION
**Pull request by jorgun**
## About the PR

makes it so the AI can view the feed from entertainment cameras
## Why / Balance

AI had no way of getting a  movable camera. 
now reporter, borgs or other can drag a camera for checking out areas AI dont have visual on
## Technical details

range is a little longer then the basic camera (7.5)
this is becuse of the microphone(range 10). and bigger lense. so its the same as a maxed out player can see.

## Media

image

## Requirements

- [x] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [x] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes

**Changelog**

:cl:
- add: AI can now view the Entertainment channels.

## Summary by CodeRabbit

- **New Features**
	- Introduced AI-driven vision capabilities to the wireless surveillance camera, enhancing its functionality.
	- Added a specified operational range of 9.5 units for improved monitoring capabilities.

### CODE REVIEW

**Comment by jorgun** [comment]
i am a bit unsure of this.
this might make the AI confiscate the cameras from the reporter.
but i think this is better then making the cameras handheld. or making the Borgs act as AI eyes.

File: Resources/Prototypes/Entities/Structures/Machines/wireless_surveillance_camera.yml [hunk 1]
```diff
@@ -37,6 +37,8 @@
       interfaces:
         enum.SurveillanceCameraSetupUiKey.Camera:
           type: SurveillanceCameraSetupBoundUi
+    - type: StationAiVision # GOOB - AI can see entertainment
+      range: 9.5
```
[the remark this answers is not in the document]
Comment (jorgun) [reply Resources/Prototypes/Entities/Structures/Machines/wireless_surveillance_camera.yml:41]: @coderabbit your drunk, thats the placement, this is a part of components.

**Review by Piras314 — APPROVED** [review]

**Comment by Aidenkrz** [comment]
Did you test this in game? 9.5 is insanely large, and you cant even do .5s with the vision comp, it just gets rounded.

**Comment by Piras314** [comment]
true the radius probably shouldn't be so large

**Comment by Aidenkrz** [comment]
> i am a bit unsure of this. this might make the AI confiscate the cameras from the reporter. but i think this is better then making the cameras handheld. or making the Borgs act as AI eyes.

ai is literally at the bottom of the totem pole of the station. they are below crew and borgs can just ignore them

**Comment by jorgun** [comment]
> Did you test this in game? 9.5 is insanely large, and you cant even do .5s with the vision comp, it just gets rounded.
deafult is 7.5 i followed current standard.

> true the radius probably shouldn't be so large
can turn it down to 7.5 same as wallmounted cameras. or you want it even lower?

**Comment by Aidenkrz** [comment]
wall mounted cameras dont have a 360 degree field also.
I'd make it 7.5 and test it, put a screenshot of it in the media part of the PR

**Comment by jorgun** [comment]
image

**Pull request merged**